### PR TITLE
fix language, punctuation, markup in sep-0006

### PR DESCRIPTION
**Pull request by TallTed**

### CODE REVIEW

File: SEP/SEP-0006/sep-0006.md [hunk 1]
```diff
@@ -128,13 +144,13 @@ is translated to:
 
 ### Evaluation
 
-To evaluate `lateral`:
+To evaluate `lateral` —
```
Comment (afs) [inline SEP/SEP-0006/sep-0006.md:148]: Please change back to colon (the style used elsewhere in this doc).

**Review by afs — APPROVED** [review]
Marked approve but there is a request about an em-dash.

File: SEP/SEP-0006/sep-0006.md [hunk 2]
```diff
@@ -128,13 +144,13 @@ is translated to:
...
```
Comment (TallTed) [inline SEP/SEP-0006/sep-0006.md:147]: Per @afs request
```suggestion
To evaluate `lateral`:
```

**Comment by afs** [comment]
Thanks you for the change. It is ready to merge.

There seems to be another matter - the IPR check is pending. But at the link it says it's acceptable!

I'll ask W3C in case there is something that will recur.

**Comment by deniak** [comment]
The initial IPR check didn't complete as expected. This should now be fixed.

**Pull request merged**